pkg/controller/controller.go

```diff
@@ -128,6 +130,7 @@ type Controller struct {
 	sentry                 sentry.Interface
 
 	name         string
+	namespace    string
```

Comment:
I'm not going to block empty string in a namespace since many operators still want to reconcile the object in all namespaces.